src/main/java/com/vmware/vipclient/i18n/base/cache/MessageCacheItem.java

```diff
@@ -31,16 +32,28 @@ public MessageCacheItem (Map<String, String> dataMap) {
 	
 	private String etag;
 	private long timestamp;
-	private Long maxAgeMillis = 864000000l;
+	private Long maxAgeMillis = 86400000l;
+	
+	private final Map<String, String> cachedData = new HashMap<String, String>();
 	
-	public final Map<String, String> cachedData = new HashMap<String, String>();
+	public void addCacheData(String key, String value) {
+		this.cachedData.put(key, value);
+	}
+	
+	public boolean isCachedDataEmpty() {
+		return this.cachedData.isEmpty();
+	}
 	
 	public synchronized void addCachedData(Map<String, String> cachedData) {
 		if (cachedData != null) 
 			this.cachedData.putAll(cachedData);
```

Comment:
If cachedData == null, then do nothing?

Reply:
Yes, do nothing because there is nothing to add.